compose/ui/ui/src/skikoTest/kotlin/androidx/compose/ui/scene/BaseComposeSceneTest.kt

```diff
@@ -78,4 +81,56 @@ class BaseComposeSceneTest {
             )
         }
     }
+
+    @Test
+    fun `cancel all pointers should cancel input coroutines`() = runTest(StandardTestDispatcher()) {
+        val scenes: List<ComposeScene> = listOf(
+            PlatformLayersComposeScene(size = IntSize(100, 100)),
+            CanvasLayersComposeScene(size = IntSize(100, 100))
+        )
+
+        scenes.forEach { scene ->
+            var cancellationsCount = 0
+            scene.setContent {
+                Box(modifier = Modifier.fillMaxSize().pointerInput(PointerEventPass.Initial) {
+                    suspendCancellableCoroutine {
+                        cancellationsCount++
+                    }
+                })
+            }
+
+            scene.sendPointerEvent(PointerEventType.Press, Offset(10f, 10f))
+            scene.sendPointerEvent(PointerEventType.Move, Offset(20f, 20f))
+            scene.cancelAllPointers()
+
+            assertEquals(1, cancellationsCount)
+        }
+    }
+
+    @Test
+    fun `cancel all pointers should cancel clicks`() = runTest(StandardTestDispatcher()) {
+        val scenes: List<ComposeScene> = listOf(
+            PlatformLayersComposeScene(size = IntSize(100, 100)),
+            CanvasLayersComposeScene(size = IntSize(100, 100))
+        )
+
+        scenes.forEach { scene ->
+            var clicksCount = 0
+            scene.setContent {
+                Box(modifier = Modifier.fillMaxSize().clickable {
+                    clicksCount++
+                })
+            }
+
+            scene.sendPointerEvent(PointerEventType.Press, Offset(10f, 10f))
+            scene.sendPointerEvent(PointerEventType.Move, Offset(20f, 20f))
+            scene.cancelAllPointers()
```

Comment:
```suggestion
            scene.cancelAllPointers()
            scene.sendPointerEvent(PointerEventType.Release, Offset(20f, 20f))
```

To properly test `clickable`, we should be sure that the gesture is really cancelled and it didn't handle Release.

If we just test `Press -> Press -> Release`, some implementations can still handle this, and we would have false positive

Reply:
Updated with more context.